derive_builder_core/src/builder.rs

```diff
@@ -80,7 +83,7 @@ impl<'a> ToTokens for Builder<'a> {
                    impl_generics);
 
             tokens.append(quote!(
-                #[derive(Default, Clone)]
+                #[derive(Default, Clone #( , #derives)* )]
```

Comment:
PS: In that case also remove `Default` and `Clone` from the unit tests - since we already have the testcase `add_derives`.